Makefile

```diff
@@ -46,44 +46,44 @@ clean: | up
 	$(call message,$(PROJECT_NAME): Removing vendor and web directories)
 	$(call docker-root, php rm -rf vendor)
 	$(call docker-root, php rm -rf web)
-	$(MAKE) -s down
+	@$(MAKE) -s down
 
 exec:
-	# Remove the first argument from the list of make commands.
+    # Remove the first argument from the list of make commands.
```

Comment:
Replaced tabs with 4 blanks - to remove it from printing into the console output and still keep it aligned in the IDE (if tab = 4 blanks)